src/ucp/core/ucp_mm.c

```diff
@@ -484,6 +517,22 @@ static ucs_status_t ucp_memh_create_from_mem(ucp_context_h context,
     return UCS_OK;
 }
 
+static UCS_F_ALWAYS_INLINE void
+ucp_memh_update_from_parent_memh(ucp_mem_h memh,
+                                 ucp_md_map_t cache_md_map,
```

Comment:
IMHO it's hard to explain meaning of those 3 md_map arguments. Maybe leave only one and make bitwise ops outside?

Reply:
added description.
`ucp_memh_update_from_parent_memh` will be used twice for `md_map` and `export_md_map`